### ISSUE TITLE
demo-category-item-test commented out

### ISSUE THREAD

**Issue by daveconnis** [code-corps/code-corps-ember#1601]
# Problem

`tests/integration/components/demo-category-item-test.js` is currently completely commented out. We need to figure out why we commented it out, and work on either removing, or uncommenting it.

## References

Progress on: #

**Comment by joshsmith** [comment]
We should remove it. The demo components are going away.

**Comment by daveconnis** [comment]
Okay, so just take away the file all together?

**Comment by joshsmith** [comment]
Yeah if it's commented out, just blow away the file. The landing page will be entirely reworked soonish.
